src/deepsparse/transformers/pipelines/question_answering.py

```diff
@@ -301,27 +287,32 @@ def process_engine_outputs(
         example = kwargs["example"]
         num_spans = len(span_extra_info)
 
-        if len(engine_outputs) != num_spans:
+        if len(engine_outputs) != 2:
```

Comment:
confirming - we now get all items across all spans back as a "batched" output from the engine step

Reply:
Yep, its now the same as every other pipeline!